module-2/backend/week_three.md

```diff
@@ -9,15 +9,40 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 
 ### Questions
 
-1. What is the entry at the command line to create a new rails app?
-2. What do Models generally inherit from in rails?
-3. What do Controllers generally inherit from in a rails project?
-4. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?
-5. What rake task is useful when looking at routes, and what information does it give you?
-6. What is an example of a route helper? When would you use them?
-7. What's the difference between what `_url` and `_path` return when combined with a routes prefix?
-8. What are strong params and why are the necessary?
-9. What role does `form_for` play in helping us create our forms?
-10. How does `form_for` know where to submit the user's input?
+1. What is the entry at the command line to create a new rails app? rails new put_name_here --skip-spring --skip-turbolinks (also can chang database being used here with --database=postgresql)
+
+2. What do Models generally inherit from in rails? Application Record
+
+3. What do Controllers generally inherit from in a rails project? ApplicationController
+
+4. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality? resources :horses, only [:show]
+
+5. What rake task is useful when looking at routes, and what information does it give you? rake routes will give you all the URI and controller path names for any given resource that you've specified so that you can call paths in a variety of ways from controllers and views
+6. What is an example of a route helper? When would you use them? these are things that rails "knows" how to call; you can write new_horses_path in a link_to of a view instead of writing the actual path like this '/horses/new'
+
+7. What's the difference between what `_url` and `_path` return when combined with a routes prefix? not sure about this one
```

Comment:
`_url` returns the absolute path (`http://...`), while `_path` returns just the root-relative path (`/horses`).